### PR TITLE
Update windows-driver-updates-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
removed unsupported versions as per https://learn.microsoft.com/en-us/windows/deployment/update/waas-manage-updates-wufb?branch=pr-en-us-8343

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3992

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/protect/windows-driver-updates-overview.md [hunk 1]
```diff
@@ -91,9 +91,6 @@ Driver updates are supported for the following Windows 10/11 editions:
 - Education
 - Pro for Workstations
 
-> [!NOTE]  
-> **Unsupported versions and editions**:  
-> *Windows 10/11 Enterprise LTSC*: Windows Update for Business (WUfB) does not support the *Long Term Service Channel* release. Plan to use alternative patching methods, like WSUS or Configuration Manager.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/windows-driver-updates-overview.md:94]:
```suggestion
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Brenduns Please review and signoff this PR. Thanks

**Comment by Brenduns** [comment]
#hold-off    @msbemba, @yogkumgit     This PR is not approved and will be closed.  

This content section was updated directly by the PM yesterday (and merged this morning for publication), as the LTSC statement he originally provided requires a more nuanced explanation. Removal of the existing statement is not the correct approach. Instead, the nuanced details for LTSC versions specific to Driver Updates is being added directly to this section.  That update is currently in the internal main branch (having been merged for publication earlier this morning from a PR built yesterday afternoon) and will update during the next doc publication cycle.

**Pull request closed without merging**